frontend/src/scenes/events/definitions/definitionDrawerLogic.ts

```diff
@@ -1,7 +1,7 @@
 import { kea } from 'kea'
 import api from 'lib/api'
 import { definitionDrawerLogicType } from './definitionDrawerLogicType'
-import { IndexedTrendResult } from 'scenes/trends/trendsLogic'
+import { IndexedTrendResult } from 'scenes/trends/types'
```

Comment:
which reminds me I had a question about when we would choose to put types in a types file and when close to point of usage?

Reply:
No clear guidelines here. This time I was just lazy: Moving the types away from `trendsLogic.ts` lets kea-typegen automatically import them inside the `trendsLogicType.ts` (avoiding circular imports). This let me write `let logic: BuiltLogic<trendsLogicType>` inside `trendsLogic.test.ts` instead of also importing and including these types as `trendsLogicType<stuff, goes, here>`.